src/views/Home.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+    <div class="layout">
+        <Layout :style="{ minHeight: '100vh' }">
+            <Sider>
+                <Menu active-name="1-2" theme="dark" width="auto">
+                    <MenuItem name="1-1">
+                        <Icon type="ios-navigate"></Icon>
+                        <span>{{ $t('views.home.option1') }}</span>
+                    </MenuItem>
+                    <MenuItem name="1-2">
+                        <Icon type="search"></Icon>
+                        <span>{{ $t('views.home.option2') }}</span>
+                    </MenuItem>
+                    <MenuItem name="1-3">
+                        <Icon type="settings"></Icon>
+                        <span>{{ $t('views.home.option3') }}</span>
+                    </MenuItem>
+                </Menu>
+            </Sider>
+            <Layout>
+                <Header
+                    :style="{
+                        background: '#fff',
+                        boxShadow: '0 2px 3px 2px rgba(0,0,0,.1)',
```

Comment:
@Andresws12 Here is possible use rem?

Reply:
@gonzalo-plaza Is temporal, for this commit, because i will change the structure and components